app/src/main/java/net/squanchy/proximity/near/NearITComponent.java

```diff
@@ -0,0 +1,15 @@
+package net.squanchy.proximity.near;
+
+import net.squanchy.injection.ActivityLifecycle;
+import net.squanchy.injection.ApplicationComponent;
+import net.squanchy.proximity.ProximityService;
+
+import dagger.Component;
+
+@ActivityLifecycle
+@Component(modules = {NearITModule.class}, dependencies = ApplicationComponent.class)
+public interface NearITComponent {
```

Comment:
This comment was actually more insightful than any dagger tutorial I stumbled on! 👍